alibi/explainers/anchor_tabular.py

```diff
@@ -128,7 +133,7 @@ def set_instance_label(self, X: np.ndarray) -> None:
              Instance to be explained.
         """
 
-        label = self.predictor(X.reshape(1, -1))[0]
+        label = self.predictor(X.reshape(1, -1))[0]  # type: int
```

Comment:
We need to declare the type here otherwise we will get warnings later on when passing `label` to functions.